File: sdk-examples/dotnet/AwsS3Example.cs
// Example: Uploading a file to S3 using AWS SDK for .NET

using Amazon.S3;
using Amazon.S3.Model;
using System;
using System.IO;
using System.Threading.Tasks;

class AwsS3Example
{
    static async Task Main()
    {
        var s3Client = new AmazonS3Client();
        var bucketName = "your-bucket-name"; // Replace with your bucket name
        var keyName = "example.txt";
        var filePath = "/path/to/example.txt"; // Replace with your file path

        using (var fileStream = new FileStream(filePath, FileMode.Open, FileAccess.Read))
        {
            var putRequest = new PutObjectRequest
            {
                BucketName = bucketName,
                Key = keyName,
                InputStream = fileStream
            };

            var response = await s3Client.PutObjectAsync(putRequest);
            Console.WriteLine("File uploaded to S3.");
        }
    }
}

File: sdk-examples/java/AwsS3Example.java
// Example: Uploading a file to S3 using AWS SDK for Java v2

import software.amazon.awssdk.services.s3.S3Client;
import software.amazon.awssdk.services.s3.model.PutObjectRequest;
import java.nio.file.Paths;

public class AwsS3Example {
    public static void main(String[] args) {
        S3Client s3 = S3Client.create();

        PutObjectRequest putRequest = PutObjectRequest.builder()
                .bucket("your-bucket-name") // Replace with your bucket name
                .key("example.txt")
                .build();

        // Upload the file located at /path/to/example.txt
        s3.putObject(putRequest, Paths.get("/path/to/example.txt"));

        System.out.println("File uploaded to S3.");

        s3.close();
    }
}